src/lib/errors.ts

```diff
@@ -0,0 +1,22 @@
+import { endsWith } from 'lodash';
+
+import { checkInt } from './validation';
+
+export function NotFoundError(err: { statusCode?: string }): boolean {
+	return checkInt(err.statusCode) === 404;
+}
+
+export function ENOENT(err: { code: string }): boolean {
+	return err.code === 'ENOENT';
+}
+
+export function EEXISTS(err: { code: string }): boolean {
+	return err.code === 'EEXISTS';
+}
+
+export function UnitNotLoadedError(err: string[]): boolean {
+	if (err[0] != null) {
```

Comment:
Since you've switched to using `_.endsWith` instead of `String.prototype.endsWith` you can just do
```typescript
export function UnitNotLoadedError(err: string[]): boolean {
	return endsWith(err[0], 'not loaded.');
}